### DISCUSSION TITLE
Client Notification After Authorization and Unauthorization

### DISCUSSION THREAD

**Discussion by nelszy** [Art-of-WiFi/UniFi-API-client#210] [Q&A]
I want my guests to be notified via email when they are authorized or unauthorized in my Unifi Controller. I have set up the SMTP server in my controller, and it works well. I also have a database that holds all the information about my guests. I am wondering if we can set some parameters in the controller using our Unifi API to accomplish my objective with PHP logics. Thanks in advance.

**Comment by malle-pietje** [comment]
AFAIK there is no API route to send email notifications.
